.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -1,6 +1,18 @@
 name: Deploy standalone plugins to WordPress.org
 
 on:
+  # TODO The pull_request will be removed once the workflow is tested.
+  pull_request:
+    branches:
+      - trunk
+      - 'release/**'
+      - 'feature/**'
+    paths:
+      - '.github/workflows/deploy-standalone-plugins.yml'
+    types:
+      - opened
+      - reopened
+      - synchronize
```

Comment:
This will be removed once testing is done. cc. @felixarntz @swissspidy @joemcgill 

Check auto workflow: https://github.com/WordPress/performance/actions/runs/7567242182/job/20606023553?pr=948

For manual workflow: Go to Actions in repo > Click on "Deploy standalone plugins to WordPress.org" in left panel > Click on "Run workflow" dropdown > Select branch > add plugin slug and check "Debug mode" checkbox > Press "Run workflow"